cmd/sys-mgmt-agent/res/configuration.toml

```diff
@@ -6,15 +6,15 @@
 # In the future, the manifest may be more dynamic or even provided by some 3rd party orchestrator.
 
 ExecutorPath = '/path/to/the/file'
```

Comment:
This line needs to be changed as well to point to the locally built Executor. See proposed example provided in the issue. Needs to be tested and verified.